src/batch.rs

```diff
@@ -291,10 +291,17 @@ impl<'a> BatchBuilder<'a> {
 
     pub fn push_complex_clip(&mut self, clip: &[ComplexClipRegion]) {
         // TODO(gw): Handle nested complex clips!
-        if clip.len() > 0 {
-            self.complex_clip = Some(clip[0]);
-        } else {
-            self.complex_clip = None;
+        self.complex_clip = match clip.len() {
+            0 => None,
+            n if n > 1 => {
```

Comment:
Probably clearer to add a condition for 1 => above this and remove the _ => condition.